hasher-matcher-actioner/hmalib/lambdas/api/matches.py

```diff
@@ -539,6 +540,13 @@ def _matches_for_hash(
 
         return match_objects
 
+    def _distance_from_match_if_exits(match: IndexMatch) -> int:
```

Comment:
blocking: Add a docstring explaining the compatibility issue and whether you think we should move to off-int distance at some future point. Feel free to shorthand to an instance.